tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Extensions/OrderExtensionsTests.cs

```diff
@@ -213,16 +189,15 @@ public static void IsSectionStatusCompleteComplete_NullOrder_ReturnsFalse()
         }
 
         [Test]
-        [CommonInlineAutoData(null, false, false, false, false, false, false)]
-        [CommonInlineAutoData(true, true, true, true, true, false, true)]
-        [CommonInlineAutoData(true, true, true, true, true, true, true)]
-        [CommonInlineAutoData(true, true, true, true, false, true, true)]
-        [CommonInlineAutoData(true, true, false, false, false, false, false)]
-        [CommonInlineAutoData(true, true, false, true, true, false, false)]
-        [CommonInlineAutoData(true, false, true, false, false, true, false)]
+        [CommonInlineAutoData(null, false, false, false, false, false)]
+        [CommonInlineAutoData(true, true, true, true, false, true)]
+        [CommonInlineAutoData(true, true, true, true, true, true)]
+        [CommonInlineAutoData(true, true, true, false, true, true)]
+        [CommonInlineAutoData(true, false, false, false, false, false)]
+        [CommonInlineAutoData(true, false, true, true, false, false)]
+        [CommonInlineAutoData(true, true, false, false, true, true)]
```

Comment:
I think the expected result should still be `false` here, as (if I remember correctly) a buyer must have viewed the solutions page before they can complete an order. Worth double-checking.